subprojects/docs/src/samples/build-organization/cross-project-output-sharing/kotlin/consumer/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+val sharedConfiguration: Configuration by configurations.creating {
+    isCanBeConsumed = false
+    isCanBeResolved = true
+}
+
+dependencies {
+    add(sharedConfiguration.name, project(mapOf("path" to ":producer", "configuration" to "sharedConfiguration")))
```

Comment:
Is this working to use this kind of declaration: https://stackoverflow.com/questions/34887720/referencing-the-outputs-of-a-task-in-another-project-in-gradle

it reduces the number of times we use the`sharedConfiguration` string.

Reply:
Yes, but there's no need for variant aware dependency resolution here. This is a simple example: Sharing one file from here to there. This sample aims to keep it simple.

Reply:
Sorry, the variant-aware part was not the main point of the message here, I should have described it better.

The syntax like this is shorted and less error prone I think, if working of course:

```kotlin
dependencies {
    sharedConfiguration(project(":projectB"))
}
```

Reply:
If:
`dependencies { sharedConfiguration(project(":projectB")) }`
works then that is a preferred over _unsafe_ maps. If that does not work there is also:
`project(path=":producer", configuration="sharedConfiguration")`
in Kotlin script.

Reply:
Thanks, I understand now. This worked and I agree it's an improvement: `sharedConfiguration(project(path = ":producer", configuration = "sharedConfiguration"))`.